src/Dockerfile

```diff
@@ -34,6 +34,10 @@ RUN apk add --no-cache \
 ADD https://ftl.pi-hole.net/macvendor.db /macvendor.db
 COPY crontab.txt /crontab.txt
 
+# Add PADD to the container, too.
+ADD --chmod=0755 https://raw.githubusercontent.com/pi-hole/PADD/PADD_FTLv6/padd.sh /usr/local/bin/padd
+RUN alias padd='port=${FTLCONF_webserver_port%%,*} padd --port ${port:-8080} --secret ${FTLCONF_webserver_api_password}'
```

Comment:
The alias is not much use here, it will need to be in start.sh like the following (tested)

```bash
# Add an alias for padd to the root user's bashrc
port="${FTLCONF_webserver_port%%,*}"
echo "alias padd='padd --port ${port:-8080} --secret ${FTLCONF_webserver_api_password}'" > /root/.bashrc
```

Another reason for it going into `start.sh` is that the `FTLCONF_` variables will be defined by the user at runtime and we don't know them when we build the image

Reply:
Thanks, I FP'd it to the right place.

Reply:
I'm sorry my suggestion was incomplete.
the complete line to put in dockerfile is:
`echo -e "alias padd='port=${FTLCONF_webserver_port%%,*} padd --port ${port:-8080}  --secret ${FTLCONF_webserver_api_password}' >>~/.bashrc"`

without the redirection to bashrc and placed in start.sh will do also.